src/keypair.rs

```diff
@@ -1,8 +1,14 @@
 use crate::*;
-use helium_crypto::{ecc608, KeyTag, KeyType, Network};
+#[cfg(feature = "ecc608")]
+use helium_crypto::ecc608;
+#[cfg(feature = "tpm")]
+use helium_crypto::tpm;
+use helium_crypto::{KeyTag, KeyType, Network};
 use http::Uri;
 use rand::rngs::OsRng;
 use serde::{de, Deserializer};
+#[cfg(feature = "ecc608")]
+use std::path::Path;
 use std::{collections::HashMap, convert::TryFrom, fmt, fs, io, path, path::Path, str::FromStr};
```

Comment:
Path isn't needed and is causing linter warning
```suggestion
use std::{collections::HashMap, convert::TryFrom, fmt, fs, io, path, str::FromStr};

```